README.md

```diff
@@ -106,6 +106,9 @@ shadow-cljs watch app
 
 # connect to REPL for the build (available while watch is running)
 shadow-cljs cljs-repl app
+
+# (if you are targeting node) connect to node-repl
+shadow-cljs node-repl app
```

Comment:
`node-repl` does not take a build argument. It is standalone.

Reply:
`cljs-repl` is used to connect to a REPL of a particular build, ie. inside a running application.

`node-repl` is standalone with nothing else loaded.